### PR TITLE
improve hpa config

### PR DESCRIPTION
**Pull request by yurymuski**
Fixed and extended example/links regarding HPA config

### CODE REVIEW

File: charts/centrifugo/values.yaml [hunk 1]
```diff
@@ -159,20 +159,27 @@ autoscaling:
   enabled: false
   minReplicas: 1
   maxReplicas: 100
-  targetCPUUtilizationPercentage: 80
-  # targetMemoryUtilizationPercentage: 80
+  cpu:
+    enabled: true
+    targetCPUUtilizationPercentage: 80
+  memory:
+    enabled: false
```
Comment (FZambia) [inline charts/centrifugo/values.yaml:166]: seems like this is a breaking change... If we add this we need to release 11.0.0. And probably make `cpu.enabled` to be `false` by default?

File: charts/centrifugo/values.yaml [hunk 2]
```diff
@@ -159,20 +159,27 @@ autoscaling:
...
 autoscalingTemplate: []
   # Custom or additional autoscaling metrics
-  # ref: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-custom-metrics
+  # ref: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-metrics-apis
+  # ref: https://github.com/kubernetes-sigs/prometheus-adapter/
   # - type: Pods
   #   pods:
   #     metric:
-  #       name: centrifugo_node_num_clients
+  #       # kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta1/namespaces/nft-preprod/pods/*/hpa_custom_metric_centrifugo_node_num_clients" | jq .
```
Comment (FZambia) [inline charts/centrifugo/values.yaml:178]: This comment contains your app specifix namespace right? Can we make it more general?

File: charts/centrifugo/values.yaml [hunk 3]
```diff
@@ -159,20 +159,27 @@ autoscaling:
...
+  #       name: hpa_custom_metric_centrifugo_node_num_clients
   #     target:
   #       type: AverageValue
-  #       averageValue: 10k
+  #       averageValue: 10000m # NOTE: # 10000m = 10 actual metric value (10 clients)
```
Comment (FZambia) [inline charts/centrifugo/values.yaml:182]: I do not understand what does it mean - 10000m value results into scaling on 10 clients?

File: charts/centrifugo/values.yaml (hunk 2 above)
Comment (yurymuski) [reply charts/centrifugo/values.yaml:178]: my bad :)

File: charts/centrifugo/values.yaml (hunk 1 above)
Comment (yurymuski) [reply charts/centrifugo/values.yaml:166]: yeah, I set cpu.enabled to true to keep no changes, but memory is a possible change, agree. I will bump version

File: charts/centrifugo/values.yaml (hunk 3 above)
Comment (yurymuski) [reply charts/centrifugo/values.yaml:182]: Thats a bit confusing, thats why I've added this NOTE comment ))

it was set correctly, you should not change to 10k =)
https://github.com/centrifugal/helm-charts/pull/62/commits/deda63bf49f1830de7f55bb9636c6d788e999872#diff-ad39959e06e750f9f1e0ac4c5ca9d67c5217087c1a205d1ff101ac062a711419L175

more info `Kubernetes-style quantities to describe metric values`
https://github.com/kubernetes-sigs/prometheus-adapter/blob/master/docs/walkthrough.md#quantity-values

Comment (yurymuski) [reply charts/centrifugo/values.yaml:182]: 1 in any metric value is 1000m for HPA config

**Comment by yurymuski** [comment]
pushed a fix

File: charts/centrifugo/values.yaml (hunk 3 above)
Comment (FZambia) [reply charts/centrifugo/values.yaml:182]: So 10000m means 10 clients right? Isn't this too small for the example then? Or I am still missing the idea?

Comment (yurymuski) [reply charts/centrifugo/values.yaml:182]: yes, you are right) 10 is just an example, we can set any)

Comment (FZambia) [reply charts/centrifugo/values.yaml:182]: Ok, got it, thx!

**Comment by FZambia** [comment]
Many thanks @yurymuski !

**Pull request merged**